CriticalMapsSnapshotTests/AppSettingsViewControllerSnapshotTests.swift

```diff
@@ -3,8 +3,7 @@
 
 @testable import CriticalMaps
 import XCTest
-
-class SettingsViewControllerSnapshotTests: XCTestCase {
+class AppSettingsViewControllerSnapshotTests: XCTestCase {
```

Comment:
When renaming the class of a SnapshotTest, there'll be new snapshot references. So we have to clean up the old references manually. There are right now old `SettingsViewControllerSnapshotTests` references in the project. Could you please clean them up?